test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -40,24 +40,25 @@ import (
 )
 
 var (
-	queries = flag.Int("queries", 100, "Number of hostname queries to make in each iteration")
-	perNode = flag.Int("per_node", 1, "Number of hostname pods per node")
-	upTo    = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
+	queriesAverage = flag.Int("queries", 10, "Number of hostname queries to make in each iteration per pod on average")
+	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
+	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
+	verbose        = flag.Bool("verbose", false, "Report extra details about operations")
```

Comment:
It would be better to use the V(3) or V(4) calls to glog rather than create your own mechanism to control verbosity. 

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/devel/logging.md